website/src/routes/api/(schemas)/string/index.mdx

```diff
@@ -2,6 +2,116 @@
 title: string
 ---
 
+import { Property } from '~/components';
+import { Link } from '@builder.io/qwik-city';
+
 # string
 
-> The content of this page is not yet ready. Until then just use the [source code](https://github.com/fabian-hiller/valibot/blob/main/library/src/schemas/string/string.ts).
+Schema that parses the input to string.
+
+```ts
+const schema = string(errorMessage, pipe);
+```
+
+## Parameter
+
+- `errorMessage` <Property type='string'/> = 'Invalid type'
+- `pipe` <Property type={{ type: 'custom', name: 'Pipe<string>' }}/> = []
+
+## Return
+
+- `schema` <Property type={{ type: 'custom', name: 'StringSchema' }}/>
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)
`StringSchema` doc is also needed, I think.
Can I write this doc in another PR?

Reply:
The question here is whether we should add this to the `string` API reference as well. Then everything would be in one place. However, it can also make the page messy.

Reply:
I have thought about it. Please add the `StringSchema` reference to this PR. You can create a new route for it. I think it is better if everything is in one PR and this PR has the goal to make the `string` API reference perfect.

Reply:
Implemented!
https://github.com/fabian-hiller/valibot/pull/220/commits/a224502ecd201223d41674a6d2138702efea9504